independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -35,8 +42,16 @@
 import io.quarkus.registry.catalog.json.JsonCatalogMerger;
 import io.quarkus.registry.union.ElementCatalog;
 import io.quarkus.registry.union.ElementCatalogBuilder;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Please remove usage of StringUtils. We don't need it anymore with Java 11 minimum version.

Reply:
will do